server/src/main/java/org/opensearch/index/translog/TranslogWriter.java

```diff
@@ -50,6 +50,7 @@
 import org.opensearch.core.Assertions;
 import org.opensearch.core.common.bytes.BytesArray;
 import org.opensearch.core.common.bytes.BytesReference;
+import org.opensearch.core.common.io.stream.BufferedChecksumStreamInput;
```

Comment:
We may no longer have to change namespaces given we aren't going with streaming uploads?

Reply:
We are still using BufferedChecksumInput for checksum calculation.